src/NHSD.BuyingCatalogue.Ordering.OrderingDatabase/Tables/SupplierContact.sql

```diff
@@ -0,0 +1,7 @@
+﻿CREATE TABLE [dbo].[SupplierContact.sql]
+(
+    [ContactId] INT NOT NULL PRIMARY KEY, 

```

Comment:
Can you move the definition of the primary key to a separate constraint so we can give it a name? Here's an example:

```sql
CONSTRAINT PK_Foobar PRIMARY KEY (ContactId)
```

The existing definition is absolutely fine functionally but the name of the primary key will be system generated (something like PK__SupplierContact__091C2AFB8DEAA962).